src/generateCSV.m

```diff
@@ -0,0 +1,13 @@
+for code = 1:5
+allFeatures = []; %lista de vetores vazia
```

Comment:
Idente melhor
Se esse comendo está dentro do 'for' acima, ele deveria estar identado.
O mesmo vale para os outros comandos deste arquivo.